### PR TITLE
fix for 6.0: not.be.visible -> not.exist

### PR DESCRIPTION
**Pull request by kuceb**
- fully backwards compatible

before merging:
- [x] remove 6.0 beta binary from package.json
- [x] remove change to .node-version (my node version manager needed a full version)

### CODE REVIEW

**Comment by Bkucera** [comment]
awesome. seems to be passing now

**Review by kevinold — APPROVED** [review]

File: cypress/tests/ui/transaction-view.spec.ts [hunk 1]
```diff
@@ -40,11 +40,10 @@ describe("Transaction View", function () {
     cy.wait("@personalTransactions");
   });
 
-  it("transactions navigation tabs are hidden on a transaction view page", function () {
+  it.only("transactions navigation tabs are hidden on a transaction view page", function () {
```
Comment (kevinold) [inline cypress/tests/ui/transaction-view.spec.ts:43]: @Bkucera Remove `.only`

**Review by kevinold — CHANGES_REQUESTED** [review]
remove `.only`

**Review by jennifer-shehane — APPROVED** [review]

**Comment by jennifer-shehane** [comment]
Going to merge this in so we can have tests runs in anticipation for 6.0 release.

**Pull request merged**